dms.py

```diff
@@ -328,22 +351,19 @@ def scheduled_event_handler(event):
 def lambda_handler(event, context):
     # Find the type of event scheduled or dms
     dms_alarms = ['dms_cpu_high', 'dms_cpu_low', 'dms_memory_high', 'dms_memory_low']
-    
+    available_metrics = ['CPUUtilization', 'FreeStorageSpace']
     message = ''
-    
     try:
         # if event is from SNS then we need to convert the message from text to json
         message = json.loads(event["Records"][0]["Sns"]["Message"])
-    except:
+    except :
         # if message is not from SNS then its from scheduled cloudwatch event and we process it directly
         message = event
-    
     if "replication_instance" in message:
         print("------cloudwatch scheduled event------")
         scheduled_event_handler(message)
-    elif message["AlarmName"] in dms_alarms:
+    elif message["AlarmName"] and message["Trigger"]["MetricName"] in available_metrics:
```

Comment:
This together with line 354 will miss memory utilization triggers. Either remove this additional check or add memory in line 354